pynestml/frontend/pynestml_frontend.py

```diff
@@ -123,6 +123,7 @@ def main():
         print('Not a valid path to model or directory: "%s"!' % FrontendConfiguration.get_path())
         return 1
     # after all argument have been collected, start the actual processing
+    sys.setrecursionlimit(10000)
```

Comment:
why is that necessary? (if it really is, then please add an explanatory comment)

Reply:
The default Python recursion limit is 1000, which might not be enough in practice if you are running an AST visitor on a deep tree, e.g. containing an automatically generated expression. I will add this as a comment.